.changesets/feat_geal_cache_docs.md

```diff
@@ -0,0 +1,12 @@
+### Preview for GraphOS Entity caching ([Issue #4478](https://github.com/apollographql/router/issues/4478))
+
+> ⚠️ This is a preview for an [Enterprise feature](https://www.apollographql.com/blog/platform/evaluating-apollo-router-understanding-free-and-open-vs-commercial-features/) of the Apollo Router. It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).
+>
+> If your organization doesn't currently have an Enterprise plan, you can test out this functionality by signing up for a free Enterprise trial.
+
+While federated GraphQL responses can be cached at the HTTP level, this is actually wasteful, as a lot of data can actually be shared between different requests. The Apollo Router now contains an entity cache, that works at the subgraph level: it is able to cache subgraph responses, splitting them by entities, and reusing entities across subgraph requests.
```

Comment:
```suggestion
The Apollo Router can now cache finer-grained subgraph responses at the entity level, which are reusable between requests.

Caching federated GraphQL responses can be done at the HTTP level, but it's inefficient because a lot of data can be shared between different requests. The Apollo Router now contains an entity cache that works at the subgraph level: it caches subgraph responses, splits them by entities, and reuses entities across subgraph requests.
```